tests/unit/channel_test.py

```diff
@@ -2,9 +2,13 @@
 
 """Tests suite for channel."""
 
+import gc
 import time
 
 import pytest
+from _service_utils import ensure_ssh_session_connected  # noqa: WPS436
```

Comment:
Those utils aren't meant to be imported directly. They only serve as helpers to the fixtures defined in `conftest.py`.
Use `ssh_client_session` and you'll get the same thing with less code.

Reply:
I can't, using `ssh_client_session` will not trigger the condition in Garbage Collector.

Reply:
I converted it to a fixture on its own.